lib/puppet/module_tool/applications/builder.rb

```diff
@@ -135,9 +135,9 @@ def write_json
           f.write(metadata.to_json)
         end
 
-        # PSON.pretty_generate is a BINARY string
+        # JSON.pretty_generate is a BINARY string
```

Comment:
PSON.pretty_generate can be ASCII_8BIT in ruby 1.9. In ruby 2.3, it's UTF-8. Also JSON.pretty_generate always returns UTF-8.